src/pow.cpp

```diff
@@ -138,6 +204,11 @@ unsigned int GetNextWorkRequired(const CBlockIndex* pindexLast, const CBlockHead
     if (pindexLast->GetBlockHash() == uint256S("0x0000000000003bd73ea13954fbbf1cf50b5384f961d142a75a3dfe106f793a20"))
         return 0x1b01c13a;
 
+    // If we are past the protocol-cleanup fork, then the minimum proof-of-work
+    // becomes something easily calculable.
+    if (protocol_cleanup)
+        nProofOfWorkLimit = 0x207fffff;
```

Comment:
Code style nit: `{}` for multi-line if statements (or move `nProofOfWorkLimit = [...]` up to same line).